libraries/redshop/src/Shipping/Tag.php

```diff
@@ -199,24 +199,30 @@ protected static function replaceShippingAddressStartEnd(&$templateHtml, $shippi
         $templateStart = explode('{shipping_address_start}', $templateHtml);
         $templateEnd   = explode('{shipping_address_end}', $templateStart[1]);
         $shippingData  = $shippingEnable ? $templateEnd[0] : '';
+        $OrderData     = \RedshopEntityOrder::getInstance($shippingAddress->order_id);
```

Comment:
could you please change the variables to lowercase with the first letter as others @Rmh1978

Reply:
Done, and also with $ShipData > $shipData.